administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -33,7 +59,116 @@ function triggerSave() {
 			$('#left-panel').toggle();
 			$('#right-panel').toggleClass('full-width');
 		});
+
+		selectedLang = $('#jform_itemlanguage').val();
+
+		if (selectedLang == '')
+		{
+			document.getElementById('target-association').src = '';
+		}
+
+		//Save button action, replacing the fault Joomla.submitbutton() with custom function
+		$('.btn-success').attr('onclick', function(){
+			var frame = $(this).attr('onclick');
+
+			frame = frame.substring(frame.lastIndexOf('(')+1,frame.lastIndexOf(')'));
+			return 'return triggerSave(' + frame + ')';
+		});
+
+		$('#reference-association').load(function (){
+			//Disable reference fields, commented out just to keep the code for ACL
```

Comment:
I've had problems before with chosen fields:

```js
$(this).contents().find('#jform_language').prop('disabled', true).trigger('liszt:updated');
```

Result:
```html
<select id="jform_language" name="jform[language]" class="chzn-done" disabled="" style="display: none;">
[...]
</select>
```

But the update is not being made, don't know why